workos/async_client.py

```diff
@@ -0,0 +1,77 @@
+from workos.audit_logs import AuditLogs
+from workos.directory_sync import DirectorySync
+from workos.organizations import Organizations
+from workos.passwordless import Passwordless
+from workos.portal import Portal
+from workos.sso import SSO
+from workos.webhooks import Webhooks
+from workos.mfa import Mfa
+from workos.events import AsyncEvents
+from workos.user_management import UserManagement
+
+from workos.utils.async_http_client import AsyncHTTPClient
+
+
+class AsyncClient(object):
```

Comment:
Instead of littering the SDK with `_events()` API methods, we'll have a separate AsyncClient, which developers can use methods from as needed.